src/ducks/context/SelectionContext.jsx

```diff
@@ -0,0 +1,70 @@
+import React, {
+  createContext,
+  useState,
+  useCallback,
+  useEffect,
+  useContext
+} from 'react'
+
+import flag from 'cozy-flags'
+
+export const SelectionContext = createContext()
+
+export const useSelectionContext = () => {
+  return useContext(SelectionContext)
+}
+
+const SelectionProvider = ({ children }) => {
+  const [isSelectionModeActive, setIsSelectionModeActive] = useState(false)
+  const [selected, setSelected] = useState([])
+  const isSelectionModeEnabled = flag('banks.selectionMode.enabled')
+
+  const activateSelectionMode = useCallback(
+    () => isSelectionModeEnabled && setIsSelectionModeActive(true),
+    [isSelectionModeEnabled]
+  )
+
+  const deactivateSelectionMode = () => setIsSelectionModeActive(false)
+
+  const addToSelection = item => {
+    setSelected(v => [...v, item])
+  }
+
+  const removeFromSelection = useCallback(
+    item => {
+      setSelected(selected.filter(e => e.id !== item.id))
+    },
+    [selected]
+  )
```

Comment:
Nit: Instead of using a list to represent a set of values, we could use an hash/object/Set ? This way unicity of values would be guaranteed and it would be a bit clearer I think.

Reply:
Not really relevant as the number of items selected will be super low.